setup1-verifier/src/verifier.rs

```diff
@@ -405,32 +320,23 @@ impl Verifier {
     /// After completion or error, the loop waits 5 seconds and starts again.
     ///
     pub async fn start_verifier(&self) {
-        // Initialize the shutdown listener
-        let verifier = self.clone();
-        tokio::task::spawn(async move {
-            let _ = verifier.shutdown_listener().await;
```

Comment:
how does the verifier go with shutting down now?

Reply:
That's a great question! As you can see, I've removed the tasks queue from the verifier. It reuses only 3 files now, and clears them when needed. If the verifier is killed there are a few options:
- it has got a task assigned, but didn't download the files to verify. In this case the assigned task will be cleared from the cache in a minute, and another verifier will pick it up
- if the verifier goes down after downloading the files, the result will be the same as in a previous case
- if the verifier is killed in the middle of uploading the files, the coordinator will produce an error and reassign the work if the verifier doesn't show up in time
- if the verifier is killed after uploading the verification, nothing happens. The coordinator will write the files to disk and call `try_verify` function